Core/include/Acts/Seeding/InternalSpacePoint.hpp

```diff
@@ -66,6 +98,9 @@ class InternalSpacePoint {
                             // for. Quality can be changed if the space point is
                             // used for a better quality seed.
   const SpacePoint& m_sp;   // external space point
+
+  DoubleMeasurementDetails
+      m_doubleMeasurement;  // double measurement details for strip coordinates
```

Comment:
It seems pretty heavy to have this attached to every single SP, even if it's not a strip SP. Is there a better way to do this? Maybe there's some kind of type-erasure mechanism that could be used.

Reply:
Another option would be to add a measurement details container, and having the InternalSpacePoint storing the pointer to the container (if any) and an integer with the location to access.
Would this be a better approach?

Reply:
I'm working on replacing the internal SP with a Delegate extractors set in the configuration, which retrieve information (e.g. covariance and x,y,z coordinates) from the external input SP. To support the functionality in this PR we could add an extra Delegate that retrieves this information from the external SP.